components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -261,140 +261,20 @@
                     }
                 },
                 "intellij": {
-                    "type": "object",
-                    "description": "Configure IntelliJ integration",
-                    "additionalProperties": false,
-                    "properties": {
-                        "plugins": {
-                            "type": "array",
-                            "description": "List of plugins which should be installed for users of this workspace. From the JetBrains Marketplace page, find a page of the required plugin, select 'Versions' tab, click any version to copy pluginId (short name such as org.rust.lang) of the plugin you want to install.",
-                            "items": {
-                                "type": "string"
-                            }
-                        },
-                        "prebuilds": {
-                            "type": "object",
-                            "description": "Enable warming up of IntelliJ in prebuilds.",
-                            "additionalProperties": false,
-                            "properties": {
-                                "version": {
-                                    "type": "string",
-                                    "enum": [
-                                        "stable",
-                                        "latest",
-                                        "both"
-                                    ],
-                                    "description": "Whether only stable, latest or both versions should be warmed up. Default is stable only."
-                                }
-                            }
-                        },
-                        "vmoptions": {
-                            "type": "string",
-                            "description": "Configure JVM options, for instance '-Xmx=4096m'."
-                        }
-                    }
+                    "$ref": "#/definitions/jetbrainsProduct",
```

Comment:
Refactored to extract a common properties to the definition and stabilize code generation.